www/src/context/SettingsContext.jsx

```diff
@@ -43,6 +45,10 @@ const SettingsContextProvider = ({ children }) => {
     }
   }, []);
 
+  useEffect(() => {
+    window.analytics.track('Toggle Settings');
```

Comment:
We should use global instead of window here. Also, I think it would make sense to also include state of `showSettings` here as well in the event properties.

Reply:
slightly changed this, to rid of `useEffect` because of initial render. Added  `value ? 'show' : 'hide'`